packages/rancher-sachet/charts/Chart.yaml

```
@@ -0,0 +1,11 @@
+annotations:
+  catalog.cattle.io/hidden: "true"
+  catalog.cattle.io/certified: rancher
+  catalog.cattle.io/release-name: rancher-sachet
+  catalog.cattle.io/os: linux
+apiVersion: v2
+name: rancher-sachet
+description: Sachet is an SMS alerting tool for the Prometheus Alertmanager https://github.com/messagebird/sachet
```

nit: keep in sync with the way it is formatted in prom2teams
```suggestion
description: A Helm chart for Sachet based on the upstream https://github.com/messagebird/sachet
```

---

fixed.